uefi.go

```diff
@@ -41,33 +45,52 @@ func (u UEFIFirmwareDevice) QemuParams(config *Config) []string {
 	return qemuParams
 }
 
+//Helper function to find paths (either code or vars firmware) for new UEFIFirmwareDevice
+func selectPath(paths []string) string {
+    for _, path := range paths {
+        if PathExists(path) {
+            return path
+        }
+    }
+    return ""
+}
+
 // NewSystemUEFIFirmwareDevice looks at the local system to collect expected
 // OVMF firmware files, callers will need to make a copy of the of the Vars
 // template file before using it in a running VM.
 func NewSystemUEFIFirmwareDevice(useSecureBoot bool) (*UEFIFirmwareDevice, error) {
 	uefiDev := UEFIFirmwareDevice{}
+	//can add in more paths as necessary
+	var SecCodePaths = []string{SecCodePath, CentosSecVars, SecCodePathAarch64}
```

Comment:
```suggestion
	var SecCodePaths = []string{SecCodePath,, SecCodePathAarch64}
```
Only Code paths here, one for x86 and one for arm64